test/simulator/misc/statuses.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
-const assert = require('./../../assert');
-const common = require('./../../common');
+const assert = require('../../assert');
+const common = require('../../common');
```

Comment:
Ditto.

I think @Zarel prefers your proposed style, but the switch is better left for a separate PR.